Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+COPY . .
```

Comment:
If you are going to copy all of . then no need to copy requirements.txt seperately, just copy it all on line 2